api/clouddrive.proto

```diff
@@ -0,0 +1,232 @@
+/// Please use the following editor setup for this file:
+// Tab size=2; Tabs as spaces; Clean up trailing whitepsace
+//
+// In vim add: au FileType proto setl sw=2 ts=2 expandtab list
+//
+// Note, the documentation provided here for can be created in
+// markdown format plus the use of 'codetabs' are supported. The documentation
+// will then be rendered by github.com/openstoreage/libopenstoreage.github.io and
+// provided on https://libopenstorage.github.io
+//
+syntax = "proto3";
+
+import "google/protobuf/timestamp.proto";
+import "google/api/annotations.proto";
+import "api/api.proto";
+
+package openstorage.api;
+
+option go_package = "api";
+option java_multiple_files = true;
+option java_package = "com.openstorage.api";
+
+// CloudDriveSet defines a set of drive pools that could be attached on a node
+message CloudDriveSet {
+	// ID is the unique id of the cloud drive. This is managed by the storage provider
+  // implementing the handlers
+	string id = 1;
```

Comment:
May be we can be explicit here and say that this id is the nodeID. This makes it clear that a single CloudDriveSet can be associated with a node.